modules/11_aerosols/exoGAINS2025/sets.gms

```diff
@@ -17,6 +17,70 @@ emisForEmiFac(all_enty)  "types of emissions that are only calculated in a post-
         NH3
 /
 
+all_sectorEmi "all sectors with emissions"
+/   indst        "emissions from industry sector"
+    res          "emissions from residential sector"
+    trans        "emissions from transport sector"
+    power        "emissions from power sector"
+    solvents     "emissions from solvents"
+    extraction   "emissions from fuel extraction"
+    indprocess   "process emissions from industry"
+    waste        "emissions from waste"
+/
+
+sectorEndoEmi(all_sectorEmi)   "sectors with endogenous emissions"
+/
+    indst    "industry"
+    res      "residential"
+    trans    "transport"
+    power    "power"
+/
+
+sectorExogEmi(all_sectorEmi) "sectors with exogenous emissions"
+/
+    solvents
+    extraction
+    indprocess
+    waste
+/
+
```

Comment:
```suggestion
```
This is not needed anymore, right? At least I did not find the set being used anywhere.